lib/tasks/mastodon.rake

```diff
@@ -256,10 +256,11 @@ namespace :mastodon do
             q.modify :strip
           end
 
-          env['SMTP_OPENSSL_VERIFY_MODE'] = prompt.ask('SMTP OpenSSL verify mode:') do |q|
-            q.required
-            q.default 'peer'
-            q.modify :strip
+          if prompt.yes?('Do you want to use a client certificate to authenticate to the server?', default: false)
+            env['SMTP_OPENSSL_VERIFY_MODE'] = 'peer'
+            prompt.ask('Path to the client certificate:') do |q|
```

Comment:
This client certificate never gets put into the environment anywhere?

Reply:
Oops, indeed. I'm going to fix it

Reply:
Hmm, actually, Mastodon does not support client certificates. Should we nuke `SMTP_OPENSSL_VERIFY_MODE`?

Reply:
Didn't you notice this because you are going to use this feature? Or were you just reviewing the code?

Reply:
I also noticed that mastodon:setup-generated SMTP config failed for me until I removed `SMTP_OPENSSL_VERIFY_MODE` from it, when setting up a new instance.

Reply:
@akihikodaki I noticed this because someone on IRC had that issue.

Reply:
Sorry for late answer:
> Hmm, actually, Mastodon does not support client certificates. Should we nuke `SMTP_OPENSSL_VERIFY_MODE`?

I think so.